SECURITY.md

```diff
@@ -11,6 +11,6 @@ receiving such patches depend on the CVSS v3.0 Rating:
 ## Reporting a Vulnerability
 
 Please responsibly disclose (suspected) security vulnerabilities to
-**[The Micronaut Foundation](foundation@micronaut.io)**. You will receive a response from
+**[The Micronaut Foundation](foundation@micronaut.io)**. You will receive a chatResponse from
```

Comment:
was this an accidental change? Don't think the security list will respond with AI